EmailServiceTestClass.cls

```diff
@@ -0,0 +1,95 @@
+/**
+* @description: Test class for fetchingEmailContents which is created to to test
+                the functionality of fetchingEmailContents.
+* @author: Noopur Naik
+*
+* @revision:
+* Version      Date            Description
+* ------------------------------------------------
+* 1.0          30-Jun-2017     Initial revision.
+**/
+@isTest
+public class EmailServiceTestClass {
+
+    static testMethod void checkForNewContactCreation()
+    {
+         Messaging.InboundEmail email = new Messaging.InboundEmail();
+         Messaging.InboundEnvelope env = new Messaging.InboundEnvelope();
+         system.Test.setMock(HttpCalloutMock.class, new MockHttpResponseGenerator());
+         email.subject = 'Test';
+         email.plainTextBody = 'Demo';
+         env.fromAddress ='test@test.com';
+         email.fromname = 'FirstName LastName';
+         
+         system.test.startTest();
+            FetchingEmailContents fetchEmail = new FetchingEmailContents();
+            fetchEmail.handleInboundEmail(email,env);
+         system.test.stopTest();
+         
+         List<contact> contactList = [Select id from contact ];
+         List<Email_Handler__c> emaiHandlerList = [Select id from Email_Handler__c ];
+         system.assertEquals(1,contactList.size());
```

Comment:
Why the assert is hardcoded? 
Why it's limited to size assert only?
Why do you check values of records like if contact is createding then you should check newly created contact should have supplied email address etc.